Neos.Eel/Classes/Utility.php

```diff
@@ -17,6 +17,18 @@
  */
 class Utility
 {
+    /**
+     * Return the expression if it is an valid EEL expression, otherwise return null.
+     *
+     * @param string $expression
+     *
+     * @return bool
+     */
+    public static function parseEelExpression($expression)
```

Comment:
This could have a return type annotation of `?string`, too

Reply:
Did not add return type to keep compatibility with PHP 7.0

Did not add parameter type hint because `evaluateEelExpression` has also none, I would prefer to change both if that is fine for you.

Reply:
As this targets master we can safely assume PHP 7.1+ (see https://www.neos.io/download-and-extend.html).

But that was a nitpick anyways.. The more I think about this the more I doubt that we can fix that class.. see discussion below